tests/components/mobile_app/test_entity.py

```diff
@@ -40,7 +41,7 @@
 
     assert entity.attributes["device_class"] == "battery"
     assert entity.attributes["icon"] == "mdi:battery"
-    assert entity.attributes["unit_of_measurement"] == "%"
+    assert entity.attributes["unit_of_measurement"] == UNIT_PERCENTAGE
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5093397)